assets/README.md

```diff
@@ -0,0 +1,36 @@
+# Mayflower-Tokens
+[![npm package][npm-badge]][npm]
+
+Mayflower-tokens provides a bundled up library of design tokens and assets that you will need for theming your front end framework to look and feel like [Mass.gov](https://mass.gov) and other Commonwealth web properties that are in Mayflower ecosystem. Mayflower-tokens includes color token SCSS variables, fonts, icons and other imagery from Mayflower — the [enterprise design system for the Commonwealth of Massachusetts][mayflower-doc].
```

Comment:
I think a text tree showing what and where the various resources live within this package could be helpful to developers using it. 

For example (probably not accurate):

```
├── fonts          
│   ├── texta          Texta web fonts
├── images             SVG icon files 
├── styles             Sass token files          
│   ├── _colors.scss   Color variables
│   └── _index.scss    All Sass variables and mixins

```

Reply:
Good point! I will include the tree